### PR TITLE
bring over dockerfile from old repo

### PR DESCRIPTION
**Pull request by martenson**
original: https://github.com/RECETOX/apLCMS/blob/master/Dockerfile

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    -e 'remotes::install_version("ROCS", version = "1.3")' \
+    -e 'remotes::install_version("snow", version = "0.4-3")'
+
+RUN Rscript -e 'remotes::install_version("BiocManager", version = "1.30.10")' 
+RUN Rscript -e 'BiocManager::install(version = "3.11",ask = FALSE)'
+RUN Rscript -e 'BiocManager::install("mzR")'
+
+RUN Rscript -e 'remotes::install_version("hdf5r", version = "1.3.2")'
+
+COPY . /apLCMS
```
Comment (martenson) [inline Dockerfile:31]: this is the only part I changed

**Comment by martenson** [comment]
If we get this to bioconda (https://github.com/RECETOX/recetox-aplcms/issues/3) we probably do not need to bother with dockerfile and dockerhub

**Comment by xtracko** [comment]
> If we get this to bioconda (#3) we probably do not need to bother with dockerfile and dockerhub

Agreed

I've also deleted r-hdf5r library form the container since it is already not part of the project.

**Review by xtracko — APPROVED** [review]

**Comment by martenson** [comment]
obsoleted by conda package
https://anaconda.org/bioconda/r-recetox-aplcms

**Pull request closed without merging**